### DISCUSSION TITLE
Could not patch WIM boot.wim

### DISCUSSION THREAD

**Discussion by ozgur-as** [ipxe/wimboot#50] [Q&A] [answered]
hello,

i am using wimboot along with ipxe to boot over network to install windows. i have successfully done so with the files from en-us_windows_10_consumer_editions_version_22h2_updated_may_2023_x64_dvd_8ae93bf4.iso with no problems, but when i apply the same procedure to the turkish version of the system (tr-tr_windows_10_consumer_editions_version_22h2_updated_may_2023_x64_dvd_0cb9bad2.iso) i get the following errors during wim patching

using 2.7.6

thanks in advance!

```
...patching WIM lookup.file at [0x28c56c2e,0x28c56c60)
Chunk 1 offset lies outside resource
Could not patch WIM boot.wim
Could not boot: Error 0x7f048281 (https://ipxe.org/7f048281)
Could not boot: Error 0x7f048281 (https://ipxe.org/7f048281)
Error occured, press any key to return to menu ...
```

**Comment by NiKiZe** [comment]
What does ifstat show if you run it before your boot command?
Also it would help to see your script.

Alternatively try with a clean iPXE boot and only initrd boot.wim followed by wimboot.

**Comment by ozgur-as** [comment]
by the way, i've also tried with 2 different isos (win11 english and turkish) and they both booted fine.

my script:
```
:boot
imgfree
kernel ${win_base_url}/configs/wimboot
initrd     ${win_base_url}/virtio/netkvm.cat   netkvm.cat
initrd     ${win_base_url}/virtio/netkvm.inf   netkvm.inf
initrd     ${win_base_url}/virtio/netkvm.sys   netkvm.sys
initrd     ${win_base_url}/virtio/netkvmco.dll netkvmco.dll
initrd     ${win_base_url}/virtio/vioscsi.cat  vioscsi.cat
initrd     ${win_base_url}/virtio/vioscsi.inf  vioscsi.inf
initrd     ${win_base_url}/virtio/vioscsi.sys  vioscsi.sys
initrd     ${win_base_url}/virtio/viostor.cat  viostor.cat
initrd     ${win_base_url}/virtio/viostor.inf  viostor.inf
initrd     ${win_base_url}/virtio/viostor.sys  viostor.sys
initrd -n winpe.jpg          ${win_base_url}/configs/winpe.jpg                winpe.jpg ||
initrd -n winre.jpg          ${win_base_url}/configs/winre.jpg                winre.jpg ||
initrd -n setup.bmp          ${win_base_url}/configs/setup.bmp               setup.bmp ||
initrd -n install.bat ${win_base_url}/configs/${win_image}.bat install.bat ||
initrd -n configure.bat ${win_base_url}/configs/configure.bat configure.bat ||
initrd -n winpeshl.ini ${win_base_url}/configs/winpeshl.ini winpeshl.ini ||
initrd -n bootmgr     ${win_base_url}/${win_image}/bootmgr       bootmgr ||
initrd -n bootmgr.efi ${win_base_url}/${win_image}/bootmgr.efi   bootmgr.efi ||      
initrd -n bcd         ${win_base_url}/${win_image}/boot/bcd      bcd ||
initrd -n boot.sdi    ${win_base_url}/${win_image}/boot/boot.sdi boot.sdi ||
initrd -n boot.wim    ${win_base_url}/${win_image}/sources/boot.wim boot.wim
boot
```

i'm attaching ifstat and only initrd boot.wim followed by wimboot output screenshots.

![clean](https://github.com/ipxe/wimboot/assets/13024600/e074c840-a449-4fa8-88cd-3658c678b311)
![ifstat](https://github.com/ipxe/wimboot/assets/13024600/49c0d0f9-7d74-4c4d-8562-9c96fe4300ab)

**Reply by NiKiZe** [reply]
My bad I meant imgstat. (Important that this runs before just before to get the correct data)
So boot.wim from 3 images works fine, but not the 10tr?
The screenshot here gives more details compared to your first message.

I would drop all bootmgr, BCD and boot.sdi lines from your script, no longer needed.

If the original name is the wanted name you do not need the name duplication.

If it still fails, try with wimboot and boot.wim only, without adding any other files at all.

**Reply by ozgur-as** [reply]
> So boot.wim from 3 images works fine, but not the 10tr? The screenshot here gives more details compared to your first message.

yup, exactly.

here is the screenshot with imgstat just before the boot line:
![dropped](https://github.com/ipxe/wimboot/assets/13024600/f9a8be1a-d90f-466b-ab6f-223941e1fe44)

i dropped all bootmgr, bcd and boot.sdi lines:
![imgstat](https://github.com/ipxe/wimboot/assets/13024600/afdcb653-5fee-4185-8701-966cb95bc487)

only wimboot and boot.wim lines:
![onlytwo](https://github.com/ipxe/wimboot/assets/13024600/c3cd862f-937e-47cd-a1df-1ff7412accfc)

**Reply by NiKiZe** [reply, marked as answer]
You still have bootmgr, BCD and boot.sdi.
But at this stage I would verify the boot.wim file. It looks on the big side, normal official is usually around 400mb?
Tried to upload a newly downloaded version to your server?

**Reply by ozgur-as** [reply]
the last screenshot didn't have them.
boot.wim for the english 22h2 is also around 600mb and the iso boots without problems with a usb stick.

anyway, i have downloaded an updated 22h2 tr iso and it booted successfully. thanks :)
